graphstorm-processing/tests/test_dist_numerical_transformation.py

```diff
@@ -114,6 +122,37 @@ def test_numerical_transformation_without_transformation(input_df: DataFrame, ch
         assert row["salary"] == expected_salary
 
 
+@pytest.mark.parametrize("norm", ["min-max", "standard", "rank-gauss"])
+@pytest.mark.parametrize("out_dtype", ["float32", "float64"])
+def test_numerical_min_max_transformation_precision(
+    spark: SparkSession, check_df_schema, out_dtype, norm
+):
+    """Test numerical transformation without any transformation applied"""
+    # Adjust the number to be an integer
+    high_precision_integer = 1.2345678901234562
```

Comment:
This is a float though? Are we trying to encode a large int perhaps?

Reply:
Yep. But we are testing are between float32 and float64. Large int is better for LongType though, so we choose a float number here.